src/main/java/io/seqera/tower/cli/utils/FilesHelper.java

```diff
@@ -39,4 +40,19 @@ public static void saveString(String fileName, String text) {
             e.printStackTrace();
         }
     }
+
+    public static String readFile(File file) {
```

Comment:
This method reads a file and deletes it always. Rename the method name to make it clear. Maybe better `readFileAndDelete` ?